integration-tests/src/test/java/io/unitycatalog/integrationtests/SparkDeltaTableCRUDTest.java

```diff
@@ -0,0 +1,135 @@
+package io.unitycatalog.integrationtests;
+
+import lombok.SneakyThrows;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.nio.file.Files;
+import java.util.List;
+import java.util.UUID;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@TestMethodOrder(OrderAnnotation.class)
+public class SparkDeltaTableCRUDTest extends BaseSparkTest {
+    private static final String BASE_TABLE_NAME = "numbers";
+    private final String RUN_ID = UUID.randomUUID().toString();
+    private static final String SCHEMA = RandomStringUtils.randomAlphabetic(8);
```

Comment:
is there a specific reason on why this has to be random?

Reply:
No strong opinion here, it just made it slightly easier for debugging (no conflicts on repeated runs if the server isn't restarted in between runs and failure prevents proper teardown)